src/DbDumper.php

```diff
@@ -272,4 +272,9 @@ protected function determineQuote(): string
     {
         return strtoupper(substr(PHP_OS, 0, 3)) === 'WIN' ? '"' : "'";
     }
+
+    protected function isWindows(): string
+    {
+        return strtoupper(substr(PHP_OS, 0, 3)) === 'WIN' ? true : false;
```

Comment:
You can remove the `? true : false` part.

Reply:
Thank you for the review. You're right.